Exercise_3.java

```diff
@@ -19,7 +24,20 @@
     void printMiddle() 
     { 
         //Write your code here
-	//Implement using Fast and slow pointers
+    	if(head == null) {
+    		return;
+    	}
+	    //Implement using Fast and slow pointers
+    	Node slow = head;
+    	Node fast = head;
+    	if (head != null) {

```

Comment:
redundant condition 
already handled in line 27

Reply:
I forgot to comment out the below condition, thanks for your input.